Sources/Just/Just.swift

```diff
@@ -726,7 +728,7 @@ extension JustOf {
 
 public final class HTTP: NSObject, URLSessionDelegate, JustAdaptor {
 
-  public init(session: FoundationNetworking.URLSession? = nil,
+  public init(session: URLSession? = nil,
```

Comment:
I'm not sure why you added `FoundationNetworking.` here. Is that because it didn't compile without the module name?

Reply:
Just confirmed that this builds on Linux. LGTM.